classList.js

```diff
@@ -12,229 +12,244 @@
 /*! @source http://purl.eligrey.com/github/classList.js/blob/master/classList.js */
 
 if ("document" in self) {
+    (function() {
```

Comment:
The enclosure should probably enclose everything.

Suggest:

```js
(function(self) {
  'use strict';

  if(!("document" in self)) {
    return;
  }
  
  // try/catch

  // ....

})(typeof self !== 'undefined' ? self : this);
```

This would help reduce nesting instead of increase it.

Such an enclosure would also fix: https://github.com/eligrey/classList.js/issues/48